### PR TITLE
Fixes the pokemon catch rates

### PR DESCRIPTION
**Pull request by Jasperrr91**
## Short Description:
This keeps track of the original pokeball catch rates and resets to those before using a second berry. Solves the problem where pokeball chances keep getting multiplied upon multiple razzberry uses.

## Fixes/Resolves/Closes (please use correct syntax):
- https://github.com/PokemonGoF/PokemonGo-Bot/issues/4836

### CODE REVIEW

File: pokemongo_bot/cell_workers/pokemon_catch_worker.py [hunk 1]
```diff
@@ -492,6 +493,7 @@ def _do_catch(self, pokemon, encounter_id, catch_rate_by_ball, is_vip=False):
                     data={'pokemon': pokemon.name}
                 )
                 used_berry = False
+                catch_rate_by_ball = original_catch_rate_by_ball
```
Comment (mjmadsen) [inline pokemongo_bot/cell_workers/pokemon_catch_worker.py]: I was literally just going to put in a PR for this. Your solution is much simpler.

**Comment by mjmadsen** [comment]
:+1:

**Comment by Jasperrr91** [comment]
Hehe, hope you didn't spent to much time on it!

**Comment by mjmadsen** [comment]
Nah, just got done and looked on here and thought to myself "oh, that's way easier..."

**Pull request merged**